config/config.json

```diff
@@ -21,7 +21,7 @@
         "EnablePostUsernameOverride": false,
         "EnablePostIconOverride": false,
         "EnableTesting": false,
-        "EnableDeveloper": false,
+        "EnableDeveloper": true,
```

Comment:
Before queuing for dev review, we'll also want to remove these changes to config.json